test/functional/test_runner.py

```diff
@@ -201,6 +201,7 @@
     'feature_backwards_compatibility.py --legacy-wallet',
     'feature_backwards_compatibility.py --descriptors',
     'wallet_txn_clone.py --mineblock',
+    'example_y2038.py',
```

Comment:
the file perms should be 755 (`sudo chmod 755 test/functional/example_y2038.py`)
```
-rwxr-xr-x 1   9070 Mar  1 22:43 example_test.py*
-rw-r--r-- 1   1301 Mar  5 18:06 example_y2038.py
```
If this is intended to be a real test, ISTM it should have another file name.

Reply:
Indeed. What would you name a test like this @jonatack assuming this approach is correct ? (unit test may be better)